packages/smooth_app/lib/l10n/app_ro.arb

```diff
@@ -428,11 +428,11 @@
             "count": {}
         }
     },
-    "plural_compare_x_products": "{count,plural, =other{Comparare {count} Produse}",
+    "plural_compare_x_products": "{suma,plural, =altceva{Comparare {count} Produse}",
```

Comment:
Uhm, I fear that this doesn't work

Reply:
I'm fixing it on Crowdin, but ideally, it should be impossible to make that mistake (removing count/plural, and putting it as a comment instead) @M123-dev

Reply:
I'm not sure what you mean @teolemon, this is needed to work and for the translators to be able to allow multiple plural forms, how should we exclude this

Reply:
Also I think there are some wrong translations in some languages due to missing comments in the past, is there some easy way to re request the translations

Reply:
adding unit tests to spot the bad ones, and then cleaning them up on Crowdin.

Reply:
Thats a good idea, looking into that